public/translation_fr.json

```diff
@@ -92,8 +92,8 @@
         "advancement": "Avancement",
         "magic_item": "Objet magique",
         "-": "-",
-        "private": "Privé",
-        "public": "Public"
+        "private": "Privée",
+        "public": "Publique"
```

Comment:
I think these refer to `Characters`. `Personnage` is `masculin` in French, therefore these should be reverted.

Reply:
I just saw Annie's comment in a previous conversation. If the translated words for `public` and `private` are below `Visibilité`, then keep what you have. I should've been more careful. Line 21 will still need to be changed to `public`.